graphql/src/query.rs

```diff
@@ -450,6 +384,22 @@ impl Mutation {
             .await?)
     }
 
+    /// Adds a bookmark to the user's list of bookmarks
```

Comment:
Does this function also "un-bookmark" stuff? If so, that should probably go in the doc string and the name should maybe change.